bdai_ros2_wrappers/bdai_ros2_wrappers/action.py

```diff
@@ -0,0 +1,583 @@
+# Copyright (c) 2024 Boston Dynamics AI Institute Inc.  All rights reserved.
+
+import collections
```

Comment:
```suggestion
from collections import deque
```

Reply:
Done in https://github.com/bdaiinstitute/ros_utilities/pull/58/commits/329efc3935f1f9fdc187d6157e4d218f4289adfd.